fdbclient/NativeAPI.actor.cpp

```diff
@@ -2429,7 +2429,8 @@ void Transaction::atomicOp(const KeyRef& key, const ValueRef& operand, MutationR
 
 	t.mutations.push_back( req.arena, MutationRef( operationType, r.begin, v ) );
 
-	if( addConflictRange )
+	// The write conflict range for a SetVersionstampedKey operation is added by the proxy at commit time.
```

Comment:
There's a similar change to be made, since WriteMap::mutate also adds a conflict range